n2y/audit.py

```diff
@@ -99,7 +100,7 @@ def main(raw_args, access_token):
         },
     }
 
-    client = Client(access_token)
+    client = Client(access_token, logger=logger)
```

Comment:
Would it be possible to just use `logger=log` here and remove the `logger=log` part of the function def in line 56?

Reply:
@russellkan, it definitely would be. I added it on line 56 so that the `main` function can be imported and run elsewhere with the option of running it with a custom logger. Otherwise, there's no point of referencing `logger` on lines 56 or 103 because the default `logger` for the `Client` class is already the package logger (i.e., `n2y.logger.logger`)

P.S. Noice pfp, my guy 🔥

Reply:
Got it. Thanks haha.